build/Palaso.proj

```diff
@@ -2,16 +2,19 @@
 	<PropertyGroup>
 		<RootDir Condition="'$(teamcity_build_checkoutDir)' == '' And '$(RootDir)'==''">$(MSBuildProjectDirectory)/..</RootDir>
 		<RootDir Condition="'$(teamcity_build_checkoutDir)' != ''">$(teamcity_build_checkoutDir)</RootDir>
+		<BuilTaskDir Condition="'$(BuilTaskDir)'==''">build</BuilTaskDir>
```

Comment:
`BuildDir`? (with `d`)
I would also consider setting the value to `$(RootDir)/build`